From 45c23dab36e6dd76023670f4c0766b6104ff0011 Mon Sep 17 00:00:00 2001
From: Artem Senichev <a.senichev@yadro.com>
Date: Tue, 13 Mar 2018 15:47:30 +0300
Subject: [PATCH] Delay for Centaure 1.2V ready on 128 DIMM system

Resolves issue with full equipped DIMM lost.
Originally created by MSI (S188).

Signed-off-by: Artem Senichev <a.senichev@yadro.com>
---
 src/usr/hwpf/hwp/dram_training/dram_training.C | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/src/usr/hwpf/hwp/dram_training/dram_training.C b/src/usr/hwpf/hwp/dram_training/dram_training.C
index bf20b7464..2d8a80e5f 100644
--- a/src/usr/hwpf/hwp/dram_training/dram_training.C
+++ b/src/usr/hwpf/hwp/dram_training/dram_training.C
@@ -329,6 +329,7 @@ void*    call_host_disable_vddr( void *io_pArgs )
 {
     errlHndl_t l_err = NULL;
     IStepError l_StepError;
+    uint8_t i;
 
     TRACDCOMP(ISTEPS_TRACE::g_trac_isteps_trace,
               ENTER_MRK"call_host_disable_vddr");
@@ -348,6 +349,12 @@ void*    call_host_disable_vddr( void *io_pArgs )
 
     }
 
+    //Delay 2 seconds for Centaure 1.2V ready on 128 DIMM system.
+    for (i = 0; i < 1; ++i)
+    {
+        fapiDelay(1000 * 1000000, 10000); // 1,000,000 ns = 1ms, 1000*1ms
+    }
+
     TRACDCOMP( ISTEPS_TRACE::g_trac_isteps_trace,
                EXIT_MRK"call_host_disable_vddr");
 
-- 
2.14.1

